src/content/content.service.ts

```diff
@@ -742,4 +819,74 @@ export class ContentService {
   hasListKeyword(queryString) {
     return queryString.toLowerCase().includes('tags');
   }
+
+  formatBenefitsListFromJobs(jobs: any[]): any[] {
```

Comment:
@milindSansare rename this function to `formatBenefitsListFromJobs` => `getFormattedEligibilityCriteriaFormBenefits`